pkg/ttl/v3/ttl.go

```diff
@@ -0,0 +1,39 @@
+package ttl
```

Comment:
the workaround makes sense to me and we have done something similar at https://github.com/envoyproxy/go-control-plane/blob/master/pkg/resource/v3/resource.go . Can you check out if both workarounds can reside together in a common folder?

Reply:
The only thing I can think of would be to do something like `pkg/per_version/vX/<file>.go`, but that seems a little bit gross too. WDYT?